app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,40 @@
+class SessionsController < ApplicationController
+  def new
+     if current_user.present?
+      flash[:error] = "You are already logged in"
+      if current_user.basic?
+        redirect_to "/profile"
+      elsif current_user.admin?
+        redirect_to "/admin"
+      elsif current_user.merchant?
+        redirect_to "/merchant"
+      end
```

Comment:
This conditional block (w/ lines 21-27) should live as a helper method but we were unsure where it should finally settle? Let's decide if this should become a user.home? method in application controller...